docusaurus/docs/themes/transportation/index.mdx

```diff
@@ -2,45 +2,129 @@ import DocCardList from '@theme/DocCardList';
-4. **Cargo**. Cargo refers to something the person or entity traveling on the transportation network is carrying. The most common example of a cargo scope is portions of the transportation network where transporting hazardous material is not allowed.
-5. **Vehicle Attributes**. These refer to specific attributes of vehicles that are too specific to infer from travel mode, or where it is overly cumbersome to generate a travel mode for each attribute combination. This includes scoping vehicles that are allowed based on attributes like height, weight, or axle count.
+<p>
 
-[View the Schema for a Connector Type](/reference/transportation/connector)
+**Scoped and Rule-Based Properties.** The Transportation theme schema allows property values
+to be specified for granular scopes at the sub-feature level. For example:
 
-[View the Schema for a Segment Type](/reference/transportation/segment)
+</p>
+
+- A speed limit on a road segment might be scoped to apply only to part of the
+  road geometry using Geometric Scoping.
+- The directionality of a segment, controlling the direction or directions in
+  which traffic can flow along the segment geometry, may be specified to change
+  at different times of day using Temporal Scoping.
+- The categories of people and vehicles who are allowed to travel on a segment
+  can be controlled using Subjective Scoping.
+- A real-world attribute that varies under changing environmental conditions can
+  be modeled using Environmental Scoping.
+
+<p>
+
+Read more about these concepts on the [Scoped and Rule-Based Properties](scoping-rules)
+page.
+
+</p>
+
+</li>
+
+<li>
+
+<p>
+
+**Roads.** The Transportation theme schema models any kind of road, street, or
+path, including dedicated walking and cycling paths, as `road` segments. Roads
+are currently the most developed part of the Transportation schema.
+
+</p>
+
+<p>
+
+Read more about roads on the [Roads](roads) page.
+
+</p>
+
+</li>
+
+<li>
+
+<p>
+
+**Travel Modes.** The Transportation theme supports a fuzzy concept called
+travel mode which can be used as a way of controlling the scope of scoped
+and rule-based properties.
+
+</p>
+
+<p>
+
+Read more about the travel modes concept and how travel modes interact
+with other scoping properties on the [Travel Modes](travel-modes) page.
+
+</p>
+
+</li>
+
+</ul>
+
+## Schema Reference
+
+- [View schema for the Connector feature type](/reference/transportation/connector)
+- [View schema for the Segment feature type](/reference/transportation/segment)
+
+## Status and Next Steps
+
+- For the initial `v0.4.0` schema release, we focused most of our efforts on
+  defining core transportations concepts and applying these concepts to roads.
+- In upcoming releases, we hope to work through our backlog of known issues and
+  ideally start work on railways and waterways and, ideally, the public
```

Comment:
Not sure it can be promised in upcoming releases.

Reply:
I will change the word "upcoming" to "future" to reduce the implication that it will be "soon" rather than "eventually"...